spec/operators/map-spec.js

```diff
@@ -88,10 +116,12 @@ describe('Observable.prototype.map()', function () {
     });
 
     expectObservable(r).toBe(expected, values);
+    expectSubscriptions(a.subscriptions).toBe(asubs);
   });
 
   it('should map with index until completed', function () {
     var a = hot('-5-^-4--3---2----1--|--8--|--#');
```

Comment:
Yes, some of RxJS 4 test does and while migrated into RxJS 5 some of them were removed those (emit-after-completion). Got curious whether we'll apply same consistency here..